torch/csrc/autograd/utils/wrap_outputs.h

```diff
@@ -54,6 +54,14 @@ inline PyObject* wrap(int64_t value) {
   return THPUtils_packInt64(value);
 }
 
+inline PyObject* wrap(uint64_t value) {
+  return THPUtils_packUInt64(value);
+}
+
+inline PyObject* wrap(void* value) {
+  return THPUtils_packUInt64(reinterpret_cast<uintptr_t>(value));
```

Comment:
Just use `intptr_t`

Reply:
It just feels weird to have negative addresses, so I kept it like this, but I can change it if you really want